### PR TITLE
Homepage ng

### LINKED ISSUE #2: Fetch API
Create APICall file, write fetch function for provided API

### LINKED ISSUE #3: Create Header for Main page
Header will have an icon, greeting banner and login/logout button

### PR DESCRIPTION
**Pull request by nicolegooden**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling- no new features

### Detailed Description / What does this PR do?

This PR creates several new files for the following `Components`:
  + Header - functional
  + MovieContainer - functional
  + MovieCard - functional
  + App (updates) - class-based

This PR creates corresponding `CSS` files for each `Component` listed above, in addition to files for the tomato image and the `apiCalls`.

This PR creates the homepage for the user to see upon first visiting the application.  The user will see a header, which includes a tomato image's alt text (currently not displaying, will fix), the title of the application, and a `log` button which will eventually serve as a login or logout button.  There is also a section for all of the movies to be displayed as a card; each movie's information has been fetched from the API with the `componentDidMount()` method in `App.js`.  

### Files modified:
[X] App.js

[X] MovieContainer.js

[X] MovieContainer.css

[X] MovieCard.js

[X] MovieCard.css

[X] Header.js

[X] Header.css

[X] apiCalls.js 

### Has This Been Tested?
- [X] No - not yet, will be tested within the next day or so.
- [ ] Yes

### Are there specific methods for testing?
Dev tools and visual inspection of the browser

### What are the relevant tickets? 

Closes #2 
Closes #3 

NOTE: We should definitely have more issues closed for this PR that don't currently exist as options on our project board.  For example, issues could be closed for the following features:

+ User can view all movies from the API
+ User can view all relevant movie details per movie on the homepage

### Message/Questions for reviewer:

Thanks for your teamwork on this!  I've looked into the distorted movie images problem, and I think we could try setting `background-image` as a property for the `.movie-card` rather than creating a separate `img` element... this is certainly an enhancement and not of high priority, but something to try out if we have the time.

### Checklist:
- [X] My code has no unused/commented out code
- [X] My code has no console logs - aside from the `console log` in the `getMovies` API call
- [X] I have performed a self-review of my own code
- [ ] I have commented on my code, particularly in hard-to-understand areas - not necessary for this one as we completed this with driver-navigator style!

### Screenshot:
Screen Shot 2020-10-06 at 6 59 08 PM

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,26 +1,34 @@
...
+    .then(allMovies => this.setState({ movies: allMovies.movies }))
+    .catch(err => {
+      this.setState({  err: 'Sorry, try again later.'});
+    })
+  }
+
+  render() {
+    return (
+      <main>
+        <Header />
```
Comment (nware1066) [inline src/App.js:27]: We might want to add some conditional logic here to display the error message if the movies fail to load

File: src/MovieContainer.css [hunk 2]
```diff
@@ -0,0 +1,5 @@
+.all-movie-cards {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  background-color: #e5e6eb;
```
Comment (nware1066) [inline src/MovieContainer.css:4]: I am glad that we got some color and styling started. I do think that we will want to revisit the color scheme

File: src/Header.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+import './tomato.jpg';
+import './Header.css';
+
+function Header() {
+  return (
+    <section className='header'>
+      <img className='tomato-image' src='./tomato.jpg' alt='basic tomato' />
```
Comment (nware1066) [inline src/Header.js:8]: Need to figure out why our tomato won't load

**Review by nware1066 — APPROVED** [review]

File: src/MovieContainer.css (hunk 2 above)
Comment (nicolegooden) [reply src/MovieContainer.css:4]: Yes, revisiting the color scheme is definitely a good idea!

File: src/Header.js (hunk 3 above)
Comment (nicolegooden) [reply src/Header.js:8]: I've added an issue for this, a `bug` label is included.

**Pull request merged**